src/apyfixedarray.h

```diff
@@ -141,6 +141,19 @@ class APyFixedArray : public APyBuffer<mp_limb_t> {
     APyFixedArray broadcast_to_python(const std::variant<nb::tuple, nb::int_> shape
     ) const;
 
+    APyFixedArray
```

Comment:
Add a short comment using "//!" on public methods, that way a docstring is generated for the C++ documentation